OpenEdXMobile/src/main/java/org/edx/mobile/http/Api.java

```diff
@@ -241,7 +241,10 @@ public EnrolledCoursesResponse getCourseById(String courseId) {
     public List<EnrolledCoursesResponse> getEnrolledCourses(boolean fetchFromCache) throws Exception {
         Bundle p = new Bundle();
         p.putString("format", "json");
-        p.putString("org", config.getOrganizationCode());
+        String org = config.getOrganizationCode();
+        if(org != null){
```

Comment:
Maybe add spaces around the condition for better formatting/readability: `if (org != null) {`